src/components/R64Button.vue

```diff
@@ -185,6 +187,9 @@ export default {
   transform: translateZ(0);
   animation: load8 1.1s infinite linear;
 }
+.loader-outline {
```

Comment:
We should extract this class to the button theme config. So people can override it.